ui/src/logic/useIsIOSWebView.ts

```diff
@@ -0,0 +1,10 @@
+import { isNativeApp } from '@/logic/native';
+
+export default function useIsIOSWebView() {
+  const userAgent = window.navigator.userAgent.toLowerCase();
+  return (
+    isNativeApp() &&
+    /(iphone|ipod|ipad).*applewebkit/.test(userAgent) &&
+    !/safari/.test(userAgent)
```

Comment:
I think we might wanna just use this one and get rid of useIsIOSSfari since it isn't used anywhere, and this one is specific to testing if we're in the native app.